src/sql/annotation/annotation.spec.ts

```diff
@@ -0,0 +1,776 @@
+          "preLimitKeyword": "",
+          "preOrderByKeyword": "
+          ",
+          "preQuery": "",
+          "preUnionKeyword": "",
+          "preWhereKeyword": "",
+        },
+        "limitKeyword": undefined,
+        "limitValue": undefined,
+        "orderByKeyword": "order by",
+        "orderBySeparators": Array [],
+        "orderByUnits": Array [
+          Object {
+            "direction": "",
+            "expression": SqlRef {
+              "innerSpacing": Object {},
+              "name": "column",
+              "namespace": undefined,
+              "namespaceQuotes": undefined,
+              "quotes": "",
+              "type": "ref",
+            },
+            "postExpression": "",
+          },
+        ],
+        "postQueryAnnotatedComments": undefined,
+        "postSelectAnnotatedComments": undefined,
+        "selectDecorator": "",
+        "selectKeyword": "Select",
+        "selectSeparators": Array [
+          Separator {
+            "left": "",
+            "right": " ",
+            "separator": ",",
+          },
+          Separator {
+            "left": "",
+            "right": " ",
+            "separator": ",",
+          },
+        ],
+        "selectValues": Array [
+          SqlRef {
+            "innerSpacing": Object {},
+            "name": "column",
+            "namespace": undefined,
+            "namespaceQuotes": undefined,
+            "quotes": "",
+            "type": "ref",
+          },
+          SqlRef {
+            "innerSpacing": Object {},
+            "name": "column1",
+            "namespace": undefined,
+            "namespaceQuotes": undefined,
+            "quotes": "",
+            "type": "ref",
+          },
+          SqlRef {
+            "innerSpacing": Object {},
+            "name": "column2",
+            "namespace": undefined,
+            "namespaceQuotes": undefined,
+            "quotes": "",
+            "type": "ref",
+          },
+        ],
+        "tableSeparators": Array [],
+        "tables": Array [
+          SqlRef {
+            "innerSpacing": Object {},
+            "name": "table",
+            "namespace": undefined,
+            "namespaceQuotes": undefined,
+            "quotes": "",
+            "type": "ref",
+          },
+        ],
+        "type": "query",
+        "unionKeyword": undefined,
+        "unionQuery": undefined,
+        "whereExpression": undefined,
+        "whereKeyword": undefined,
+        "withKeyword": undefined,
+        "withSeparators": undefined,
+        "withUnits": undefined,
+      }
+    `);
+
+    expect(parser(sql).toString()).toMatchInlineSnapshot(`
+      "Select column, column1, column2 from table
+          order by column"
+    `);
+  });
+});
+
+describe('Queries with annotated comments post select and post query', () => {
+  it('single annotated comment comment', () => {
+    const sql = `Select column, column1, column2 from table 
+    --: valueName = value
```

Comment:
I do not think there should be annotations here, instead they should be at the end of individual select parts like:

```sql
Select
  column --: v=1
, column1 --: v=2
, column2 --: v=3
from table
```